tests/unit/amo/containers/TestApp.js

```diff
@@ -189,6 +195,24 @@ describe('App', () => {
     expect(root.find(NestedStatus)).toHaveProp('code', 200);
   });
 
+  it('configures a default HTML title for Firefox', () => {
+    const { store } = dispatchClientMetadata({ clientApp: CLIENT_APP_FIREFOX });
+    const root = shallowUntilTarget(
```

Comment:
even though it's only used in two places I think it would be nice to make a helper function for this like `shallowRender()` or something.

Reply:
ah that makes sense, thanks! 💯